runtime/executor/src/test/java/org/apache/nemo/runtime/executor/data/DirectByteBufferOutputStreamTest.java

```diff
@@ -33,71 +34,79 @@
  */
 public class DirectByteBufferOutputStreamTest {
   private DirectByteBufferOutputStream outputStream;
+  private static final MemoryPoolAssigner memoryPoolAssigner = new MemoryPoolAssigner(1, 32);
 
   @Before
-  public void setup(){
-    outputStream = new DirectByteBufferOutputStream();
+  public void setup() throws MemoryAllocationException {
+    outputStream = new DirectByteBufferOutputStream(memoryPoolAssigner);
   }
 
   @Test
-  public void testSingleWrite() {
+  public void testSingleWrite() throws IOException, IllegalAccessException {
     int value = 1;
     outputStream.write(value);
     assertEquals(value, outputStream.toByteArray()[0]);
+    outputStream.release();
```

Comment:
Seeing this made me wonder whether DirectByteBufferOutputStream#release() can be replaced with DirectByteBufferOutputStream#close(), which is empty at the moment.

I'll think more about this.